scripts/variantstore/docs/aou/AOU_DELIVERABLES.md

```diff
@@ -57,9 +57,9 @@
    - Run if there are any samples to withdraw from the last callset.
 1. **TBD Workflow to soft delete samples**
 1. `GvsPopulateAltAllele` workflow
-   - **TODO:** needs to be made cumulative so that it can add data to the existing table instead of creating it from scratch on each run (see [VS-52](https://broadworkbench.atlassian.net/browse/VS-52))
    - This step loads data into the `alt_allele` table from the `vet_*` tables in preparation for running the filtering step.
    - This workflow does not use the Terra Data Entity Model to run, so be sure to select the `Run workflow with inputs defined by file paths` workflow submission option.
+   - This step is cumulative (as are all of the steps prior), and it is the last step that is--so be sure that all samples have been loaded or withdrawn before progressing to the next step
```

Comment:
if data is loaded into the alt allele table for a specific sample, and then that exact sample is withdrawn later...how do we adjust the alt allele table to be sure that the sample is not used in the filter model?